examples/app_commands/basic.py

```diff
@@ -68,4 +68,30 @@ async def joined(interaction: discord.Interaction, member: Optional[discord.Memb
     await interaction.response.send_message(f'{member} joined in {member.joined_at}')
 
 
+# A Context Menu is an app command that can be run on a member or on a message by rightclicking.
+# It always takes an interaction as its first parameter and a Member, Message or Union of both as its second parameter.
```

Comment:
```suggestion
# It always takes an interaction as its first parameter and a Member or Message as its second parameter.
```

The union note is incorrect, it supports `Union[discord.User, discord.Member]`, but this is pretty niche usage so no point in mentioning it since it makes the sentence awkward.

Reply:
Oh, my bad. Thanks fore making this clear lol